docs/alerts/monitors/create-monitor.md

```diff
@@ -176,17 +176,22 @@ Sumo Logic automatically resolves the incident when the resolution condition is
 | Threshold type | How you want the value compared. Select either greater than, greater than or equal, less than or equal, or less than. |
 | Threshold | The value against which the resolution will be evaluated. You can specify any valid numeric value. |
 
-**Alert and recovery window**
+#### Alert and recovery window
 
-This setting controls the behavior of alert/recovery for sparse/intermittent/incomplete data sources.
+This setting affects both the alert generation logic and the alert recovery logic.
 
 ![metrics alert datapoints.png](/img/monitors/minimum-datapoints.png)
 
-`Alert and recovery require a minimum of <Count> data points for “at all times” evaluation windows`
+`Alert and recovery require a minimum of <Count> data points for `at all times` evaluation windows`
```

Comment:
```suggestion
`Alert and recovery require a minimum of <Count> data points for "at all times" evaluation windows`
```

This must be double quotes to match the UI (see screenshot below)
<img width="601" alt="Screenshot 2023-02-07 at 12 57 42 PM" src="https://user-images.githubusercontent.com/56411016/217364202-bbbcdc67-ab44-4bf4-a8c7-c9892e6a7b15.png">